scripts/upgrade_lido_locator.py

```diff
@@ -0,0 +1,42 @@
+import time
+import os
+
+from brownie import interface, accounts
+from utils.shapella_upgrade import get_tx_params
+from utils.config import (
+    prompt_bool,
+    get_deployer_account,
+    lido_dao_template_address,
+    lido_dao_lido_locator_implementation,
+    contracts,
+)
+
+
+def main():
+
```

Comment:
Please consider increasing the max fee from `100 gwei` to `300 gwei`.
The network is busy this week.